lib/elixir/lib/gen_server.ex

```diff
@@ -775,6 +807,7 @@ defmodule GenServer do
   list of `{key, value}` tuples representing the current process dictionary of
   the `GenServer` and `state` is the current state of the `GenServer`.
   """
+  @doc deprecated: "Use format_status/1 callback instead"
   @callback format_status(reason, pdict_and_state :: list) :: term
             when reason: :normal | :terminate
```

Comment:
I wonder if we should still document it. In any case, let's also add a TODO to remove it on 2.0?

Reply:
Let's add the TODO and ship it!